Use new gRPC default port

jufab opened the pull request:
<!--
We appreciate your contribution to the OpenTelemetry project! 👋🎉

Before creating a pull request, please make sure:
- Your PR is solving one problem
- Please provide enough information so that others can review your pull request
- You have read the guide for contributing
  - See https://github.com/open-telemetry/opentelemetry-js/blob/master/CONTRIBUTING.md
- You signed all your commits (otherwise we won't be able to merge the PR)
  - See https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md#sign-the-cla
- You added unit tests for the new functionality
- You mention in the PR description which issue it is addressing, e.g. "Fixes #xxx". This will auto-close
  the issue that your PR fixes (if such)
-->

## Which problem is this PR solving?

New Default gRPC OTEL Default port.

## Short description of the changes

Use new gRPC Otel default port => https://github.com/open-telemetry/opentelemetry-specification/pull/1221

vmarchaud reviewed (commented):
Did you test that our collector exporter is compatible with the 0.16 version of the collector ?

jufab commented:
@vmarchaud I used examples/collector-exporter-node to test my modification and I update collector version (0.16.0): 
- I modified tracing.js and metrics.js to use grpc exporter
- I modified in node_modules (@opentelemetry/exporter-collector-grpc ) exporter with new default port (4317)
- After launch docker, metrics.js and tracing.js, I verified in prometheus and zipkin.

And everything is OK.

obecny reviewed (commented):
But this change according to spec should be also applied to the rest of collectors as port `55680` is gone
Please update also each README.md with information of last used version of collector (it is in top of the readme)

jufab commented:
@obecny It's done.

dyladan reviewed (approved):
Please fix the build

dyladan commented:
@jufab if you fix the lint I can merge this

jufab commented:
> @jufab if you fix the lint I can merge this

I have a problem with the linter.

_Example_

this error : 
<img width="880" alt="lint-error" src="https://user-images.githubusercontent.com/25395032/102800287-f5625000-43b3-11eb-99e0-07a9ec677186.png">

I fix it manually

<img width="612" alt="manual-fix" src="https://user-images.githubusercontent.com/25395032/102800321-feebb800-43b3-11eb-97c8-6d00b8cb313d.png">

I run npm run lint:fix, and after that, I have

<img width="612" alt="autofix" src="https://user-images.githubusercontent.com/25395032/102800394-17f46900-43b4-11eb-8f5f-6095705a1697.png">

What's wrong with linter? I have no idea. 
Can you help me?

dyladan commented:
You probably have an out-of-date version of the linting dependencies. I would do `lerna clean -y` and `lerna bootstrap` again and see if that fixes it.

jufab commented:
I tried without success so I reinstalled all.
😊
